backend/src/modules/lobby/lobbyRoutes.js

```diff
@@ -194,45 +99,58 @@ router.post(
       });
     });
 
-    res.json({ lobbyId: lobbyResponse.insertId });
+    res.json({ lobbyId });
   },
 );
 
-router.get('/:lobbyId', (req, res, next) => {
-  const { lobbyId } = req.params;
-  res.json({});
-});
-
-router.get('/:lobbyId/fetchDraft', async (req, res, next) => {
+router.get('/:lobbyId', async (req, res, next) => {
   const { lobbyId } = req.params;
 
   const dbConnection = req[DB_CONNECTION_KEY];
 
-  const dbResponseNumber = await dbConnection.query(
-    `SELECT COUNT(user_id) AS playersCount FROM lobby_user WHERE lobby_id = ?;`,
+  const dbResponsePlayers = await dbConnection.query(
+    `SELECT user_id FROM lobby_user WHERE draft_order IS NULL and lobby_id = ?;`,
     [lobbyId],
   );
 
-  const { playersCount: playersCount } = dbResponseNumber[0];
+  const draftStarted = dbResponsePlayers.length === 0;
+
+  res.json({ draftStarted: false });
+});
+
```

Comment:
co dělá toto přesně? Je to je dočasné nebo to je nějaký magic?

Reply:
Dal jsem na podminku pro zobrazeni buttonu pro vstup na draft page - draft musi zacit/mit vylosovany poradi. Nevim, jeslti zustane nebo ne, takze se pripadne odstrani.